components/benefit_sponsors/app/models/benefit_sponsors/benefit_sponsorships/benefit_sponsorship.rb

```diff
@@ -386,6 +386,24 @@ def benefit_applications_by(ids)
       benefit_applications.find(ids)
     end
 
+    # Open enrollment can be extended only for recent applications (i.e, upto 6 months old)
+    # Benefit Application's with overlapping coverage can't be extended
+    def oe_extendable_benefit_applications
```

Comment:
Now allowing OE extension on current month and future benefit applications only.